cypress/page/index.ts

```diff
@@ -5,3 +5,4 @@ export { LoginPage } from "./login.page"
 export { AddressStepPage } from "./address-step.page"
 export { PaymentStepPage } from "./payment-step.page"
 export { ShippingStepPage } from "./shipping-step.page"
+export { DressesListPage } from "./dresses-list.page"
```

Comment:
To avoid this warning:
![image](https://user-images.githubusercontent.com/10209180/180073148-20260c31-7770-4e0f-831a-e17fe4f0a5dd.png)

You should add an empty line at the end of the file (or linebreak), that line is called EOF. You can configure your visual studio code to add automatically the EOF: https://stackoverflow.com/a/44704969 

Next time that we see this, we will comment just EOF as a reminder 😊